feature-detects/indexeddb.js

```diff
@@ -16,7 +16,13 @@ define(['Modernizr', 'prefixed'], function(Modernizr, prefixed) {
   // - Firefox shipped moz_indexedDB before FF4b9, but since then has been mozIndexedDB
   // For speed, we don't test the legacy (and beta-only) indexedDB
 
-  var indexeddb = prefixed('indexedDB', window);
+  var indexeddb;
+  try {
+    indexeddb = prefixed('indexedDB', window);
+  } catch (e) {
+    indexeddb = null;
```

Comment:
sorry for not catching before, but this line is unnecessary. can you remove it in both files and squash one more time?